Project.toml

```diff
@@ -13,6 +13,7 @@ ForwardDiff = "f6369f11-7733-5829-9624-2563aa707210"
 Interpolations = "a98d9a8b-a2ab-59e6-89dd-64a1c18fca59"
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
 Optim = "429524aa-4258-5aef-a3af-852621145aeb"
+Plots = "91a5bcdd-55d7-5caf-9e0b-520d859cae80"
```

Comment:
We should not add this as a dependency. We use `Requires.jl` to only load plotting functionality if `Plots.jl` is loaded by user. Do you have any plot functionality within this PR?

Reply:
No, the plot functionality is not required in this PR. The dependency of Plots.jl has been removed.